presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/APIFileHandler.java

```diff
@@ -77,15 +79,15 @@ public Response replaceFileFromURL(String path, String url)
         try {
             AgentFileUtils.replaceFile(Paths.get(baseDir.toString(), path), url);
             LOGGER.debug("Successfully replaced file '%s' with url '%s'", path, url);
-            return Response.status(Status.ACCEPTED).build();
+            return Response.status(Status.ACCEPTED).entity("Successfully replaced file with url").build();
```

Comment:
`ACCEPTED` responses generally should say that they acknowledge the request, not that it worked.

Though here the problem is that we're sending back `ACCEPTED` instead of something more appropriate for a synchronous request, like `OK`, in which case the body might be something like "Set property '{file}/{property}' to '{value}'". This comes up for quite a few of the responses changed in this PR.